docker-dev/omsserviceregistry/Dockerfile.stage

```diff
@@ -0,0 +1,13 @@
+FROM node:8
```

Comment:
@linuxbandit It's pretty stable AFAIK and also there are some benefits instead of using v7 (speed improvements of both node and npm, async/await, util.promisify etc.), so I think it would be better with Node v8. That's what I was going to do with oms-events actually, because I want to use some new fancy features like async/await, it makes code way more readable.